ToopherSecondFactor/toopher-openam.js

```diff
@@ -1,183 +1,84 @@
 // vim: ts=2 sw=2 expandtab cindent
```

Comment:
It complects the code by adding another concept that is orthogonal to Toopher's JavaScript extension for OpenAM. Taken to the extreme, we could include directives for Emacs, Atom, Sublime, TextMate, WebStorm, Eclipse, NetBeans, Visual Studio, Aptana, Ace, Komodo, Notepad++, etc to every file we edit; I think we'd rather err on the side of being editor agnostic.